src/rules/index.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
-  rules: [require("./no-default-alt-text"), require("./no-generic-link-text")],
+  rules: [
+    require("./no-default-alt-text"),
+    require("./no-generic-link-text"),
+    require("./no-empty-string-alt"),
```

Comment:
🤔 Maybe we should call this one `no-empty-alt` or `no-empty-alt-text`?

Reply:
I like `no-empty-alt-text`!

Reply:
Renamed in https://github.com/github/markdownlint-github/pull/85/commits/f2d977498194b03bcd4a6c1cf1264834f148dcf3.